### ISSUE TITLE
Feature: 3d vew

### ISSUE THREAD

**Issue by realcodywburns** [chemicstry/recksplorer#4]
Feature request: Feeding the graph data into something like ngraph.pixel to give a cleaner picture. 
https://github.com/anvaka/ngraph.pixel#demo

**Comment by tyzbit** [comment]
You might be able to just drop in the code from [here](https://bl.ocks.org/tyzbit/e41587d241b0ab0f38273dc4b8dd571e).  Or contact me on the LND slack for another 3D visualizer that shows capacities, average channel sizes, and counts the channels for each node.

**Comment by chemicstry** [comment]
Yeah, I could easily add other ngraph visualizers too, but all the highlighting and text features are hard to port. That's why I'm still hesitant to do it. Current view also needs a rewrite on webgl or canvas due to svg lag. I'm currently very busy so there is no ETA on that, but anyone is welcome to do a PR.
